src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/ClaimantRejectionStaffNotificationService.java

```diff
@@ -58,6 +63,29 @@ public void notifyStaffClaimantRejectPartAdmission(Claim claim) {
         );
     }
 
+    public void notifyStaffWithClaimantsIntentionToProceed(Claim claim) {
+        requireNonNull(claim);
+
+        EmailContent emailContent = claimantDirectionsHearingContentProvider.createContent(getParameters(claim));
+
+        emailService.sendEmail(
+            staffEmailProperties.getSender(),
+            EmailData.builder()
+                .to(staffEmailProperties.getRecipient())
+                .subject(emailContent.getSubject())
+                .message(emailContent.getBody())
+                .build()
+        );
+    }
+
+    public static Map<String, Object> getParameters(Claim claim) {
+        return new ImmutableMap.Builder<String, Object>()
+            .put("claimReferenceNumber", claim.getReferenceNumber())
```

Comment:
can we use constant already defined for these keys

Reply:
done